tfe.go

```diff
@@ -32,6 +32,7 @@ const (
 	DefaultAddress = "https://app.terraform.io"
 	// DefaultBasePath on which the API is served.
 	DefaultBasePath = "/api/v2/"
+	PingEndpoint    = "ping"
```

Comment:
Maybe we should add a small comment to explain what this is used for?

Reply:
Uuh... Did you test how this works against a PTFE instance that doesn't offer the `/ping` endpoint? Or did that endpoint already exist? If not it would be good to double check if the required headers are returned in that case (I think they are not returned correctly when getting a 301 or 302).

Reply:
> Maybe we should add a small comment to explain what this is used for?

👍

> Did you test how this works against a PTFE instance that doesn't offer the `/ping` endpoint?

It returns a 404, just like the previous request to `/api/v2/` did. The headers are there.